scripts/convert.test.js

```diff
@@ -0,0 +1,70 @@
+const mockCsvData = `Cities,"Population, All persons, People","Population, persons, Annual Growth %"
+Greater Sydney,4920000,0.021
+Greater Perth,2040000,0.016`;
+
+const mockExpectedOutput = `[
+  {
+    "id": "greater-sydney",
+    "shortName": "Sydney",
+    "description": "Description of Sydney",
+    "name": "Greater Sydney",
+    "indices": {
+      "population": 4920000,
+      "growthRate": 0.021
+    }
+  },
+  {
+    "id": "greater-perth",
+    "shortName": "Perth",
+    "description": "Description of Perth",
+    "name": "Greater Perth",
+    "indices": {
+      "population": 2040000,
+      "growthRate": 0.016
+    }
+  }
+]`;
+
+const mockActualOutput = [];
+const originalWrite = process.stdout.write;
+
+jest.mock('get-stdin', () => () => {
+  return new Promise(resolve => {
+    process.stdout.write = (str) => mockActualOutput.push(str);
+    resolve(mockCsvData);
+  });
+});
+
+jest.mock('../src/constants', () => ({
+  INDICATORS: {
+    population: {
+      name: 'The population indicator name',
+      source: 'Population, All persons, People',
+    },
+    growthRate: {
+      name: 'Annual population growth rate',
+      source: 'Population, persons, Annual Growth %',
+    },
+  },
+  CITIES: {
+    'Greater Sydney': {
+      id: 'greater-sydney',
+      shortName: 'Sydney',
+      description: 'Description of Sydney',
+    },
+    'Greater Perth': {
+      id: 'greater-perth',
+      shortName: 'Perth',
+      description: 'Description of Perth',
+    },
+  },
+}));
+
+const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));
+
+it('Converts the data correctly', async () => {
+  require('./convert');
+  await sleep(1); // This horrific line is because I can't `await` running the script directly
```

Comment:
For future reference... using `done()`  _should_ work, but doesn't because of a bug in Jest...
```js
it('Converts the data correctly', done => {
  require('./convert');

  setTimeout(() => {
    expect(process.stdout.write).toHaveBeenCalledWith(mockExpectedOutput);
    done();
  }, 100);
});
```